### PR TITLE
initial payment reversals commit

### PR DESCRIPTION
**Pull request by ps2-controller**
A few notes
- new payment reversals endpoint; takes the original line item ID; @dangdennis -- would love your feedback on the naming of the route
- @goodgoodwish --- note the updated field to the `line_item` schema: `linked_line_item` 
- Also, in the middleware, the new payment reversal route will call your retroactive event processing function passing in all the relevant info to make a single payment reversal.

### CODE REVIEW

File: schemas/LineItem.json [hunk 1]
```diff
@@ -85,6 +85,11 @@
           "description": "A description of this particular line item if any. More common for adjustments.",
           "example": "Refund for invalid purchase",
           "nullable": true
+        },
```
Comment (ps2-controller) [inline schemas/LineItem.json:88]: @ps2-controller <> @goodgoodwish --- to discuss linking to the original line item; API interface structure later

**Comment by ps2-controller** [comment]
@vikram-canopy -- anything left in the MW before moving this to RC?

**Comment by vikram-canopy** [comment]
> @vikram-canopy -- anything left in the MW before moving this to RC?

all good now, merging

File: openapi.json [hunk 2]
```diff
@@ -980,6 +980,63 @@
         }
       }
     },
+    "/accounts/{account_id}/line_items/payment_reversals/{line_item_id}": {
+      "post": {
+        "tags": ["line_items"],
+        "summary": "Create a payment for a specific account",
```
Comment (dangdennis) [inline openapi.json:986]: This summary is incorrect

Comment (vikram-canopy) [reply openapi.json:986]: good catch

**Review by dangdennis — APPROVED** [review]

**Comment by dangdennis** [comment]
Ready to merge!

File: openapi.json (hunk 2 above)
Comment (vikram-canopy) [reply openapi.json]: fixed

Comment (vikram-canopy) [reply openapi.json]: ah so did you lol

**Pull request merged**